libs/common/src/sql-db-entity/azure-file-info.ts

```diff
@@ -0,0 +1,50 @@
+import { Column, Entity, JoinColumn, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UtilisationReportEntity } from './utilisation-report';
+
+@Entity('AzureFileInfo')
+export class AzureFileInfoEntity {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  /**
+   * The folder that the file is contained within in Azure Storage
+   * @example '123'
+   */
+  @Column()
+  folder!: string;
+
+  /**
+   * The filename
+   * @example '2023_12_Bank_utilisation_report.csv'
+   */
+  @Column()
+  filename!: string;
+
+  /**
+   * The full path of the file in Azure Storage (i.e. the folder and filename)
+   * @example '123/2023_12_Bank_utilisation_report.csv'
+   */
+  @Column()
+  fullPath!: string;
+
+  /**
+   * The URL used to directly access the file in Azure Storage
+   * @example 'https://my-storage.file.core.windows.net/file-share-name/123/2023_12_Bank_utilisation_report.csv'
+   */
+  @Column()
+  url!: string;
+
+  /**
+   * MIME type - indicating the nature and format of the document
+   * @example 'text/csv'
+   */
+  @Column()
+  mimetype!: string;
+
+  /**
+   * The associated utilisation report
+   */
+  @OneToOne(() => UtilisationReportEntity, (report) => report.azureFileInfo)
```

Comment:
Do we want this relation to be bi-directional?

Reply:
FWIW I don't think so. Also are bi-directional relationships supported in TFM for one to one? Can imagine they are just only seen them for many to one/many where a join table exists